calculator.js

```diff
@@ -3,24 +3,35 @@ const numTwo = document.getElementById("num-two")
 const equalsBtn = document.getElementById("equals-btn");
 const clearBtn = document.getElementById("clear");
 const digits = document.getElementById("digits");
+const operation = document.getElementById("operation")
 
-//Click event for the clear button
 clearBtn.addEventListener("click", function (event) {
     event.preventDefault();
     numOne.value = "";
     numTwo.value = "";
     digits.innerText = ""
 });
 
-//Click event for the equals button
 equalsBtn.addEventListener("click", function (event) {
     event.preventDefault();
 
-    //Checks if entries are numbers then displays answer
     if ((isNaN(numOne.value)) || (isNaN(numTwo.value))) {
         digits.innerText = "Error";
-    } else {
+    } else if (numOne.value === "" && numTwo.value === "") {
+        digits.innerText = "Error";
+    } else if (numOne.value === "" || numTwo.value === "") {
+        digits.innerText = "Error";
+    } else if (operation.value === "+") {
         let total = parseFloat(numOne.value) + parseFloat(numTwo.value);
         digits.innerText = total;
```

Comment:
What an amazing job! This is a lot of code and it definitely works and I am positive it really helped give you a better understanding of what exactly you were doing.  An easier alternative with far less code would have also been a simple switch statement. I love that you are seeing the value of commits. There is really no such thing as too many commits. My current project that I am working on I have 127 commits on it and I probably could have done a few more! Keep up the awesome work and thank you for turning in early.

Reply:
Thanks Carrye! Yes, maybe I will convert it to switch statement at some point, that is a good idea.